tally_ho/apps/tally/views/data/tally_list_view.py

```diff
@@ -20,6 +21,12 @@ class TallyListDataView(LoginRequiredMixin,
         'administer',
         'edit',
     )
+    order_columns = (
```

Comment:
Are these used anywhere?

Reply:
Yes, the django datatables package use's these columns when sorting/ordering columns.

Reply:
Cool